src/udiskssimplejob.c

```diff
@@ -109,8 +109,10 @@ udisks_simple_job_complete (UDisksSimpleJob     *job,
                             gboolean             success,
                             const gchar         *message)
 {
+  static const gchar none[] = "";
+
   g_return_if_fail (UDISKS_IS_SIMPLE_JOB (job));
-  udisks_job_emit_completed (UDISKS_JOB (job), success, message);
+  udisks_job_emit_completed (UDISKS_JOB (job), success, (message) ? message : none);
```

Comment:
Why do we need the ``none`` "array"? To me it looks like ``message ? message : ""`` should do just fine.